docs/source/developer/architecture/backend/mantid_snapper.rst

```diff
@@ -0,0 +1,40 @@
+Mantid Snapper
+==============
+The Mantid Snapper class is a wrapper around the :term:`Mantid` Algorithm API that allows for meta processes to be performed around a queue of algorithms.
+Examples may include: Progress reporting, Quality of Life improvements, multi-threading, retrieving algorithm output properties, etc.
```

Comment:
What is meant by "Quality of Life improvements"?

Reply:
I would consider something like CloneWorkspace a QoL algorithm

Reply:
Oh gotcha, this makes sense. Thanks for the answer!